exchanges/hitbtc/hitbtc_wrapper.go

```diff
@@ -185,9 +184,8 @@ func (e *Exchange) FetchTradablePairs(ctx context.Context, _ asset.Item) (curren
 
 	pairs := make([]currency.Pair, len(symbols))
 	for x := range symbols {
-		index := strings.Index(symbols[x].ID, symbols[x].QuoteCurrency)
 		var pair currency.Pair
-		pair, err = currency.NewPairFromStrings(symbols[x].ID[:index], symbols[x].ID[index:])
+		pair, err = currency.NewPairFromStrings(symbols[x].BaseCurrency, symbols[x].QuoteCurrency)
```

Comment:
So what was causing this was:

```
{ID:COAIUSDT BaseCurrency:COAI QuoteCurrency:BTC QuantityIncrement:0.001 TickSize:0.001 TakeLiquidityRate:0.0025 ProvideLiquidityRate:0.0012 FeeCurrency:BTC}
```

Which now converts COAIUSDT -> COAI-BTC which is an invalid symbol. QuoteCurrency looks to be a settlement currency? So there are a lot of pairs now converting quote from USDT -> USD. Which is also invalid.

So I suggest instead of using quote currency we roll back and use Base currency in our index and then handle the potential -1 issue. 

```go
index := strings.Index(symbols[x].ID, symbols[x].BaseCurrency)
```

suggestion: Also a driveby here would be to handle the param asset type to ensure that asset.Spot was passed in.

Reply:
Thanks for the suggestions.
I don't think there's any value on indexing the symbol by the base currency... it should be the first character, right?
But we can just use trim prefix.
Added a test to make sure we get sanity back out.
Wanted to make sure that the USDT pairs are coming out correctly, but couldn't see a way to do that and ... this'll do.

Fixed https://github.com/gbjk/gocryptotrader/commit/ac5f4336a and https://github.com/gbjk/gocryptotrader/commit/9b50bea1f